x-pack/plugins/ml/public/services/ml_api_service/data_frame.js

```diff
@@ -60,7 +60,7 @@ export const dataFrame = {
   },
   stopDataFrameTransformsJob(jobId) {
     return http({
-      url: `${basePath}/_data_frame/transforms/${jobId}/_stop`,
+      url: `${basePath}/_data_frame/transforms/${jobId}/_stop?force=true`,
```

Comment:
looks like the force argument is being ignored in the server route

Reply:
oh of course, thanks for catching this! I updated it in 37dd905